internal/commands/resolve.go

```diff
@@ -136,15 +217,26 @@ func createResolveChannelAttachment(inc model.Incident, userName string) slack.A
 				Title: "Description",
 				Value: inc.DescriptionResolved,
 			},
+			{
+				Title: "Post Mortem meeting link",
+				Value: postMortemMeetingURL,
+			},
 		},
 	}
 }
 
-func createResolvePrivateAttachment(inc model.Incident) slack.Attachment {
+func createResolvePrivateAttachment(inc model.Incident, postMortemMeetingURL string) slack.Attachment {
 	var privateText strings.Builder
 	privateText.WriteString("The Incident <#" + inc.ChannelId + "> has been resolved by you\n\n")
 	privateText.WriteString("*Status.io:* Be sure to update the incident status on" + inc.StatusPageUrl + "\n")
-	privateText.WriteString("*Post Mortem:* Don't forget to bookmark Post Mortem for the incident <#" + inc.ChannelId + ">\n\n")
+	privateText.WriteString("*Post Mortem:* Don't forget to bookmark Post Mortem for the incident <#" + inc.ChannelId + ">\n")
```

Comment:
This message remembers the commander to bookmark the meeting, so to create a new event.
As this meeting is not obligatory now, you can check if that is a URL, if so inform it, if not, only remember to fill in the postmortem document.